### PR TITLE
slightly improved SV hunter APL

### PR DESCRIPTION
**Pull request by Meatwookie**

### CODE REVIEW

**Comment by Zuliane** [comment]
remove those two lines:
 actions.moknathal+=/raptor_strike,if=talent.serpent_sting.enabled&dot.serpent_sting.remains<gcd
  actions.moknathal+=/raptor_strike,if=buff.moknathal_tactics.remains<4&buff.mongoose_fury.stack=6&buff.mongoose_fury.remains>=gcd

don't remove the condition of "actions.precombat+=/explosive_trap,if=!talent.steel_trap.enabled"
casting both steel_trap and explosive_trap precombat isn't really doable without boss movement

we should also change orc racial usage to:
actions+=/blood_fury,if=buff.spitting_cobra.up|!talent.spitting_cobra.enabled&buff.aspect_of_the_eagle.up

don't change the spitting_cobra lines. we don't want to increase its interval, since it is our highest damage per execute. it goes up from 62.3sec to 64.6sec.

**Comment by Meatwookie** [comment]
I thought I deleted the unnecessary raptor strikes and the change in orc racial wasn't intentional.
However the prepull actions are possible without boss movement. Explosion trap has a way slower travel time then steel trap

**Comment by Zuliane** [comment]
The orc racial was my fault. ET/Str prepull seems to be somewhat working after a quick test so I guess it's fine.

File: engine/class_modules/sc_hunter.cpp [hunk 1]
```diff
@@ -6444,7 +6444,7 @@ void hunter_t::apl_surv()
   action_priority_list_t* moknathal    = get_action_priority_list( "moknathal" );
   action_priority_list_t* nomok        = get_action_priority_list( "nomok" );
 
-  precombat -> add_action( "explosive_trap,if=!talent.steel_trap.enabled" );
+  precombat -> add_action( "explosive_trap,if" );
```
Comment (nuoHep) [inline engine/class_modules/sc_hunter.cpp:6447]: stray ",if"

File: engine/class_modules/sc_hunter.cpp [hunk 2]
```diff
@@ -6461,34 +6461,36 @@ void hunter_t::apl_surv()
   moknathal -> add_action( "raptor_strike,if=buff.moknathal_tactics.stack<=1" );
   moknathal -> add_action( "raptor_strike,if=buff.moknathal_tactics.remains<gcd" );
   moknathal -> add_action( "snake_hunter,if=cooldown.mongoose_bite.charges<=0&buff.mongoose_fury.remains>3*gcd&time>15" );
-  moknathal -> add_action( "a_murder_of_crows,if=focus>55&buff.mongoose_fury.stack<4&buff.mongoose_fury.duration>=gcd" );
-  moknathal -> add_action( "flanking_strike,if=cooldown.mongoose_bite.charges<=1&buff.aspect_of_the_eagle.remains>=gcd|cooldown.mongoose_bite.charges<=1&buff.aspect_of_the_eagle.down" );
-  moknathal -> add_action( "lacerate,if=(focus>60&buff.mongoose_fury.duration>=gcd&refreshable&cooldown.mongoose_bite.charges=0&buff.mongoose_fury.stack<3)|(buff.mongoose_fury.down&cooldown.mongoose_bite.charges<3&refreshable)" );
+  moknathal -> add_action( "a_murder_of_crows,if=focus>60&-buff.moknathal_tactics.remains*focus.regen&buff.mongoose_fury.stack<4&buff.mongoose_fury.duration>=gcd" );
```
Comment (nuoHep) [inline engine/class_modules/sc_hunter.cpp:6464]: stray "&"

**Comment by nuoHep** [comment]
Also, either drop the commits to profiles (and I'll regenerate them in a single commit after merge) or prepare them by "simming" the appropriate "generate" simc files instead of hand-editing the profiles.

**Pull request closed without merging**